src/net_processing.cpp

```diff
@@ -1591,7 +1593,9 @@ void PeerManagerImpl::NewPoWValidBlock(const CBlockIndex *pindex, const std::sha
 
             LogPrint(BCLog::NET, "%s sending header-and-ids %s to peer=%d\n", "PeerManager::NewPoWValidBlock",
                     hashBlock.ToString(), pnode->GetId());
-            m_connman.PushMessage(pnode, msgMaker.Make(NetMsgType::CMPCTBLOCK, *pcmpctblock));
+
+            const CSerializedNetMsg& ser_cpmptblock{lazy_ser.get()};
```

Comment:
This is a typo I guess?
```suggestion
            const CSerializedNetMsg& ser_cmpctblock{lazy_ser.get()};
```

Reply:
Thx, fixed